ownCloudAppShared/Client/Collection Views/CollectionViewController.swift

```diff
@@ -167,10 +167,14 @@ open class CollectionViewController: UIViewController, UICollectionViewDelegate,
 	public override func loadView() {
 		super.loadView()
 
+		let secureView = SecureTextField().secureContainerView
```

Comment:
The suggestions about unifying the code to inject a secure container that I made in `BrowserNavigationViewController.swift` would also apply here.